File: answers/4.graphics/graphics.py
import numpy as np
import matplotlib.pyplot as plt

# 1. Create dummy data
data = np.random.rand(10, 10)

# 2. Generate the heatmap
plt.imshow(data, cmap='viridis')
plt.title('Python Heatmap')
plt.colorbar(label='Value')
plt.show()

File: answers/4.graphics/graphics.jl
using Plots
using Random

# Set plotting backend (e.g., pyplot(), gr(), plotly())
pyplot() 

# 1. Create dummy data
Random.seed!(42)
data = rand(10, 10)

# 2. Generate the heatmap
heatmap(data, 
        title = "Julia Heatmap", 
        c = :viridis, 
        aspect_ratio = 1)
